src/core/mesa.py

```diff
@@ -1,14 +1,20 @@
+from .pilha import Pilha
+
+
 class Mesa:
-    def __init__(self, pilhas: list, monte: Pilha):
-        pass
+    def __init__(self):
+        print("Instanciou Mesa")
+        self._pilhas = [Pilha() for _ in range(8)]
+        self._monte = Pilha()
+        self._monte.instancia_baralho()
```

Comment:
aqui o baralho não tá sendo retornado em lugar nenhum. eu mudaria os diagramas e definiria para ele ser instanciado por mesa e talvez ser um atributo de mesa até, mas podemos discutir...mesmo que não for feito dessa forma ele tem que ser retornado em algum lugar

Reply:
Na defesa oral 2 ele reclamou disso, "Baralho" agora é criado por uma função de "Mesa", "instanciar_baralho()".

Mas não vejo necessidade de ter um atributo "baralho" para "Mesa" até por que nesse caso teríamos que adicionar uma relação de agregação no diagrama de classes, certo?

Reply:
De qualquer forma, fiz as alterações correspondentes à nova versão do Diagrama de Sequência UML, talvez a pergunta acima fuja um pouco do escopo do "Inicializar" per se.